theano/tensor/elemwise.py

```diff
@@ -504,7 +504,7 @@ def __setstate__(self, d):
         self.nfunc = None
         self.inplace_pattern = frozendict(self.inplace_pattern)
         if getattr(self, 'nfunc_spec', None):
-            self.nfunc = getattr(numpy, self.nfunc_spec[0])
+            self.nfunc = getattr(np, self.nfunc_spec[0])
         elif 0 < self.scalar_op.nin < 32:
             self.ufunc = np.frompyfunc(self.scalar_op.impl,
                                           self.scalar_op.nin,
```

Comment:
Travis is failing with the following error. http://pastebin.com/au4MJiWV .  Fix all these `flake8` error messages. To ensure you have fixed them, run, `theano-nose theano/tests/test_flake8.py` locally. Also, it would be nice if commit messages are distinguishable.
Thanks